script/copy-dict.js

```diff
@@ -0,0 +1,48 @@
+// This script will copy all `.d.ts` files from `lib/` into `dist/`
+
+var fs = require('fs')
+var path = require('path')
+
+var rootDir = path.resolve(__dirname, '..')
+var distDir = path.join(rootDir, 'dist')
+var libDir = path.join(rootDir, 'lib')
+
+function* walk(dir) {
+  for (var d of fs.readdirSync(dir)) {
+    var entry = path.join(dir, d)
+    var stat = fs.lstatSync(entry)
+    if (stat.isDirectory()) yield* walk(entry)
+    else if (stat.isFile()) yield entry
+  }
+}
+
+function copy(libFilePath) {
+  var distFilePath = path.join(distDir, path.relative(libDir, libFilePath))
+  var dir = path.dirname(distFilePath)
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir, {recursive: true}) // Need Node.js 10.12.0+
+  }
```

Comment:
```suggestion
  fs.mkdirSync(dir, {recursive: true}) // Need Node.js 10.12.0+
```

I don’t think it’s needed to check?